cli/dcoscli/marathon/main.py

```diff
@@ -32,11 +32,17 @@ def _main(argv):
         argv=argv,
         version='dcos-marathon version {}'.format(dcoscli.version))
 
-    return cmds.execute(_cmds(), args)
+    marathon_client = marathon.create_client()
+    pod_subcommand = MarathonPodSubcommand(resource_reader=_get_resource,
+                                           marathon_client=marathon_client)
```

Comment:
I don't think we should move these to main. This require the user to have `dcos_url` or `marathon_url` configured always- even for commands like `help` and `info` and `config-schema`.